adapters/nobid/nobid.go

```diff
@@ -0,0 +1,133 @@
+package nobid
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// NoBidAdapter - NoBid Adapter definition
+type NoBidAdapter struct {
+	endpoint string
+}
+
+// NewNoBidBidder Initializes the Bidder
+func NewNoBidBidder(endpoint string) *NoBidAdapter {
+	return &NoBidAdapter{
+		endpoint: endpoint,
+	}
+}
+
+type nobidParams struct {
+	SiteID string `json:"SiteID"`
+}
```

Comment:
This struct is unused. Did you intent to parse the params for any purpose? Either way, this is already defined by the openrtb_ext.ExtImpNoBid struct.

Reply:
Fixed